Cargo.toml

```diff
@@ -103,8 +103,8 @@ zipf = "5.0.1"
 bitflags = "1.0.1"
 fail = "0.3"
 uuid = { version = "0.6", features = [ "serde", "v4" ] }
-grpcio = { version = "0.4", features = [ "secure" ] }
-raft = "0.4.3"
+grpcio = { version = "0.5.0-alpha.3", features = [ "secure" ] }
+raft = "0.6.0-alpha"
```

Comment:
Does it compatible with 0.4.x, in particular, the membership change?

Reply:
Yes, old APIs still  work.